scipy/special/tests/test_basic.py

```diff
@@ -113,6 +113,29 @@ def binom_int(n, k):
                           nk,
                           atol=0, rtol=0)
 
+    def test_binom_nooverflow_8346(selfself):
+        # Test (binom(n, k) doesn't overflow prematurely */
+        dataset = [
+            [1000, 500, 2.70288240945436551e+299],
+            [1002, 501, 1.08007396880791225e+300],
+            [1004, 502, 4.31599279169058121e+300],
+            [1006, 503, 1.72468101616263781e+301],
+            [1008, 504, 6.89188009236419153e+301],
+            [1010, 505, 2.75402257948335448e+302],
+            [1012, 506, 1.10052048531923757e+303],
+            [1014, 507, 4.39774063758732849e+303],
+            [1016, 508, 1.75736486108312519e+304],
+            [1018, 509, 7.02255427788423734e+304],
+            [1020, 510, 2.80626776829962255e+305],
+            [1022, 511, 1.12140876377061240e+306],
+            [1024, 512, 4.48125455209897109e+306],
+            [1026, 513, 1.79075474304149900e+307],
+            [1028, 514, 7.15605105487789676e+307]
+        ]
+        dataset = np.asarray(dataset)
+        FuncData(cephes.binom, dataset, (0, 1), 2, rtol=1e-12).check()
+
+
```

Comment:
Pep8 checker complains about two empty lines where it should have been one. Can you fix this please. Also while you're at it, what's the selfself argument, can change it back to a usual `self'.

Reply:
(Also use a list of tuples over a list of lists. It looks prettier.)